### ISSUE TITLE
Improve document localization process?

### ISSUE THREAD

**Issue by preco21** [electron/electron#7585]
Hello all!

Since `Electron` provides multi-language documents, many of people are contributing documents translations to `Electron`.

But the translating process seems to be painful.

Here is my basic translating / updating process:

1. Checking for document updates from `upstream`.
2. Find last translated changes.
3. Compare changes via [Github Compare](https://help.github.com/articles/comparing-commits-across-time/) from last translated changes to recent changes.
4. Create new branch from forked `Electron` repo.
5. Translate documents same as `upstream`.
6. Commit changes.
7. PR to `upstream`.

So, each time to translate documents, I follow this process each time.

As you can see, this kind of translating process is too complex and hard work to translator. Also there is no translator friendly tool for reviewing changes. (Even above process is not efficient, I still think the process should be improved)

I feel need improved doumenting process.

Basically, there is few known solutions to solve localization issues, although personally recommend [Crowdin](https://crowdin.com). Crowdin provides cool features for Markdown based translation. And also support diff by default, notify to translator for new updates, great UI, and reviewing tools. You can refer [here](https://github.com/thejameskyle/crowdin-sync), which is guide for syncing Crowdin documents to git repo.

How about this, any another good ideas?

**Comment by kevinsawicki** [comment]
So using Crowdin, we would keep the english docs in this repo still and then sync them up to Crowdin and then people (like yourself) would translate them in that tool?

**Comment by preco21** [comment]
@kevinsawicki Correct. [Babel Handbook](https://github.com/thejameskyle/babel-handbook) is good example to go.

**Comment by zcbenz** [comment]
I'm merging this to #2524.
